File: VideoWeb/VideoWeb.EventHub/Models/ParticipantHeartbeat.cs
using System;

namespace VideoWeb.EventHub.Models
{
    public class ParticipantHeartbeat
    {
        public Guid ConferenceId { get; set; }
        public Guid ParticipantId { get; set; }
        public HeartbeatHealth HeartbeatHealth { get; set; }
    }
}

File: VideoWeb/VideoWeb/ClientApp/src/app/services/models/participant-heartbeat.ts
export class ParticipantHeartbeat {
    constructor(
        public conferenceId: string,
        public participantId: string,
        public heartbeatHealth: HeartbeatHealth,
        public browserName: string,
        public browserVersion: string,
        public osName: string,
        public osVersion: string
    ) {}
}

export enum HeartbeatHealth {
    None = 'None',
    Good = 'Good',
    Poor = 'Poor',
    Bad = 'Bad'
}
